src/components/index.js

```diff
@@ -1,9 +1,14 @@
 import '@mdi/font/css/materialdesignicons.min.css';
 import 'vuetify/dist/vuetify.min.css';
+
 import Authentication from './Authentication/';
+import Breadcrumb from './Breadcrumb.vue';
+import FileBrowser from './FileBrowser.vue';
 import Upload from './Upload.vue';
 
 export {
-  Upload,
   Authentication,
+  Breadcrumb,
```

Comment:
My vote would be that we do not expose this component in index.js, at least not at this time. It seems like it's more of an internal helper component for some of our other components, rather than something that will stand alone.

I could be convinced otherwise, if people feel differently. @jeffbaumes @jbeezley @subdavis

Reply:
Yes, I would agree minimizing the api surface until we have a need to expose it.